index.html

```diff
@@ -31,5 +31,11 @@ <h4 class="text feature-title"> No internet needed. </h4>
         </div>
       </div>
     </div>
+    <div class="footer">
+      <h4 class="subscription-title font white-font"> Get info on new updates </h4>
+      <input class="subscription-input rounded-input" type="text">
+      <button class="subscribe-button font white-font" type="button"> Subscribe </button>
```

Comment:
the type of this button should be `submit` to make this the form's submit button.